pkg/gofr/http/multipart_file_bind.go

```diff
@@ -1,14 +1,30 @@
 package http
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
 	"io"
 	"mime/multipart"
 	"reflect"
 	"strconv"
+	"strings"
 
 	"gofr.dev/pkg/gofr/file"
 )
 
+var (
+	errNilInterface             = errors.New("cannot set value on a nil interface")
+	errUnsupportedInterfaceType = errors.New("unsupported interface value type")
+	errDataLengthExceeds        = errors.New("data length exceeds array capacity")
+	errUnsupportedKind          = errors.New("unsupported kind")
+	errSettingValue             = errors.New("error setting value at index")
```

Comment:
The error name is not consistent with the others. It's about something that failed

Something like this would sound better

```suggestion
	errCannotSetValue             = errors.New("error setting value at index")
```


```suggestion
	errSettingValueFailure             = errors.New("error setting value at index")
```

Reply:
updated error name to  `errSettingValueFailure`